platforms/macos/src/context.rs

```diff
@@ -5,13 +5,11 @@
 
 use accesskit::NodeId;
 use accesskit_consumer::Tree;
-use objc2::{
-    foundation::MainThreadMarker,
-    rc::{Id, Shared, WeakId},
-};
+use icrate::{AppKit::*, Foundation::MainThreadMarker};
```

Comment:
Nit: `AppKit::*` is a _lot_ of types, I'd probably recommend importing the specifics you need instead (when feasible)